lib/main/atom/onSaveHandler.ts

```diff
@@ -6,7 +6,8 @@ import atomConfig = require('./atomConfig'); ///ts:import:generated
 ///ts:import=parent
 import parent = require('../../worker/parent'); ///ts:import:generated
 
-import {errorView, show} from "./views/mainPanelView";
+import {getFileStatus} from "../atomts";
```

Comment:
This will create a cyclic reference. I'll fix it post merge, but just wanted to point that out :rose: